pages/Hypr Ecosystem/hyprpaper.md

```diff
@@ -5,6 +5,81 @@ title: hyprpaper
 
 hyprpaper is a fast, IPC-controlled wallpaper utility for Hyprland.
 
+## Installation
+
+{{% details title="Arch" closed="true" %}}
+
+```sh
+pacman -S hyprpaper
+```
+
+{{% /details %}}
+
+{{% details title="OpenSuse" closed="true" %}}
+
+```sh
+zypper install hyprpaper
+```
+
+{{% /details %}}
+
+{{% details title="Fedora" closed="true" %}}
+
+```sh
+sudo dnf install hyprpaper
+```
+
+{{% /details %}}
+
+
+{{% details title="Manual" closed="true" %}}
+### Dependencies
```

Comment:
```suggestion

### Dependencies

```

In the same manner as above, `{{% directives %}}` should be surrounded by empty lines, otherwise formatting breaks.
I also prefer headings to have the separation for better legibility when editing.